homeassistant/components/xiaomi_miio/air_quality.py

```diff
@@ -41,52 +47,54 @@
 
```

Comment:
Please remove `temperature` and `humidity` from `PROP_TO_ATTR`.

Reply:
see comment down below